boot/arch/arm64/meson.build

```diff
@@ -26,14 +26,32 @@
 # THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #
 
-ARCH_SOURCES = \
-	arch/$(UARCH)/src/entryjmp.S \
-	arch/$(UARCH)/src/fibril.S \
-	arch/$(UARCH)/src/stacktrace.c \
-	arch/$(UARCH)/src/stacktrace_asm.S \
-	arch/$(UARCH)/src/syscall.c \
-	arch/$(UARCH)/src/tls.c \
-	arch/$(UARCH)/src/thread_entry.S
+BUILD = true
 
-ARCH_AUTOCHECK_HEADERS = \
-	arch/$(UARCH)/include/libarch/fibril_context.h
+POST_OUTPUT = 'image.iso'
+POSTBUILD = 'grub'
+GRUB_LOADER = 'chainloader'
+
+# FIXME: The loader crashed without explanation when CONFIG_COMPRESSED_INIT is disabled.
```

Comment:
Should there be a ticket for this instead?

Reply:
Probably, yes.